src/test/kotlin/com/vanniktech/rxriddles/solutions/Riddle37Solution.kt

```diff
@@ -0,0 +1,11 @@
+package com.vanniktech.rxriddles.solutions
+
+import io.reactivex.Observable
+import java.io.IOException
+
+object Riddle37Solution {
+  fun solve(source: Observable<Boolean>) =
```

Comment:
Can you put the `=` on it's own line with 6 spaces and reintend the rest?

Reply:
Not sure having `=` on its own line make sense. I believe assignment operator should never be first character of a line and indenting the expression body based on the `=` would introduce more indentation than just using braces (`{}`). So I would suggest going with braces over a new line (if current style is unacceptable)